bundles/org.openhab.binding.androidtv/src/main/feature/feature.xml

```diff
@@ -4,6 +4,7 @@
 
 	<feature name="openhab-binding-androidtv" description="AndroidTV Binding" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
+		<feature>openhab-transport-upnp</feature>
```

Comment:
If I'm correct, this is allready a dependency for openhab-runtime-base, so it should not be needed.

Reply:
I thought that was just mdns that was removed recently?  upnp I thought was still required to be pulled in.

Reply:
You are right, i mixed them up.